app/assets/javascripts/live_search.js

```diff
@@ -219,30 +216,36 @@
     }
   };
 
+  LiveSearch.prototype.updateSortOptions = function updateSortOptions(results, action) {
+    if(action !== $.param(this.state)) { return }
```

Comment:
What is the reason 'action' is passed here?

Reply:
We want to ensure that the action is in sync with the state.

If you make a request and then make another request we only want the second request to update the DOM. This ensures that happens by tracking the state at the time of the request with the current state.